compose/cli/main.py

```diff
@@ -1065,9 +1066,11 @@ def up(self, options):
         remove_orphans = options['--remove-orphans']
         detached = options.get('--detach')
         no_start = options.get('--no-start')
+        attach_dependencies = options.get('--attach-dependencies')
 
         if detached and (cascade_stop or exit_value_from):
```

Comment:
Here, shouldn't you test with `attach_dependencies` in the `or` section?

Reply:
👍 - fixed. I've also added it in a subsequent section, as it's not compatible with `--no-start`.